mergify_engine/queue/merge_train.py

```diff
@@ -280,6 +286,44 @@ async def create_pull(self) -> None:
 
         await self.update_summaries(check_api.Conclusion.PENDING)
 
+    def _generate_merge_queue_pull_body(
+        self,
+        queue_rule: typing.Optional[rules.EvaluatedQueueRule] = None,
+    ) -> str:
+        description = f"""
+{self._get_embarked_refs(markdown=True)} are embarked together for merge.
+
+This pull request has been created by Mergify to speculatively check the mergeability of #{self.user_pull_request_number}.
+You don't need to do anything, Mergify will close it automatically when your check-runs have finished.
+"""
+        if queue_rule is not None:
+            description += "\n\nRequired conditions for merge:\n"
+            for cond in queue_rule.conditions:
+                checked = " " if cond in queue_rule.missing_conditions else "X"
+                description += f"\n- [{checked}] `{cond}`"
+
+        description += "\n\n**The following pull requests are queued:**\n"
+
+        for queue_name, grouped_cars in itertools.groupby(
+            self.train._iter_pseudo_cars(), key=lambda c: c.config["name"]
+        ):
+            formatted_cars = ", ".join(
+                (f"#{car.user_pull_request_number}" for car in grouped_cars)
+            )
+            description += f"\n* {queue_name}: {formatted_cars}"
+
+        description += "\n\n**Speculative merge train 🚂🚃🚃🚃**\n* "
```

Comment:
so far we avoided to talk about trains… do we continue? :)

Reply:
What about:

<hr />

Branch **master** (102d9de8434e98ea34dbbcf71750a7e970b59167), #1010, #1011 are embarked together for merge.

This pull request has been created by Mergify to speculatively check the mergeability of #1011.
You don't need to do anything. Mergify will close this pull request automatically when it is complete.


Required conditions for merge:

- [X] `status-success=continuous-integration/fake-ci`

**The following pull requests are queued:**

* default: 
  * feat(sub): make subscription retrieval asynchronous #1010 (checked in place)
  * feat(worker): re-add threading but without concurrents module #1011 (speculativily checked at #1020)
  * feat(sources): just send what we need for the engine #1012
  * Pull request for less-celery-tasks #1013
 
* low: 
  * chore: no more celery task for commands #1014
  * chore: remove obslete celery tasks  #1015
  
---



More informations about Mergify merge queue can be found in the [documentation](https://docs.mergify.io/actions/queue.html).

<details>
<summary>Mergify commands</summary>

<br />

You can also trigger Mergify actions by commenting on this pull request:

- `@Mergifyio refresh` will re-evaluate the queue rules

Additionally, on Mergify [dashboard](https://dashboard.mergify.io/) you can:

- look at your merge queues
- generate the Mergify configuration with the config editor.

Finally, you can contact us on https://mergify.io/
</details>
<hr />
:sparkling_heart:&nbsp;&nbsp;Mergify is proud to provide this service for free to open source projects.

:rocket:&nbsp;&nbsp;You can help us by [becoming a sponsor](/sponsors/Mergifyio)!

Reply:
1. Not sure we need to talk about *every* queues there since this PR should be in only one queue, right?
2. By the way, do we check we only put a PR in only one queue at a time? How do we report the error if so?
3. ❤️  for adding the PR title. In general, I think it's a huge win for usability. We should do it more often. I'd rather use something like a table to make things clearer, with the position in the queue.

Otherwise I do like this better, so 👍

Reply:
>     1. Not sure we need to talk about _every_ queues there since this PR should be in only one queue, right?

I think if people didn't see the whole picture of what will be merged everywhere it will thought it's PR is not queued for merge.

>     2. By the way, do we check we only put a PR in only one queue at a time? How do we report the error if so?

Yes, this is done by the heart of the engine, you can't run two queue actions, only the first that match is ran. A explicit check-runs is created say the action is ignored because another rule already run it.

>     3. heart  for adding the PR title. In general, I think it's a huge win for usability. We should do it more often. I'd rather use something like a table to make things clearer, with the position in the queue.

I will try something.